d2l-table.html

```diff
@@ -0,0 +1,282 @@
+			:host ::content > tbody > tr[selected] th {
+				border-top-color: var(--d2l-table-row-border-color-selected) !important;
+			}
+
+			:host ::content > tbody > tr[selected] + tr > td,
+			:host ::content > tbody > tr[selected] + tr > th {
+				border-top-color: var(--d2l-table-row-border-color-selected) !important;
+			}
+
+			:host ::content > tbody > tr[active][selected],
+			:host([selectable]) ::content > tbody > tr[selected]:hover {
+				background-color: var(--d2l-table-row-background-color-active-selected);
+			}
+
+			:host-context([dir="rtl"]) ::content > tbody > tr[active][selected] >:first-child,
+			:host-context([dir="rtl"]):host([selectable]) ::content > tbody > tr[selected]:hover >:first-child,
+			:host ::content > tbody > tr[active][selected] >:last-child,
+			:host([selectable]) ::content > tbody > tr[selected]:hover >:last-child {
+				border-right-color: var(--d2l-table-row-border-color-active-selected) !important;
+				border-left-color: var(--d2l-table-border-color);
+			}
+
+			:host-context([dir="rtl"]) ::content > tbody > tr[active][selected] >:last-child,
+			:host-context([dir="rtl"]):host([selectable]) ::content > tbody > tr[selected]:hover >:last-child,
+			:host ::content > tbody > tr[active][selected] >:first-child,
+			:host([selectable]) ::content > tbody > tr[selected]:hover >:first-child {
+				border-left-color: var(--d2l-table-row-border-color-active-selected) !important;
+				border-right-color: var(--d2l-table-border-color);
+			}
+
+			:host ::content > tbody > tr[active][selected] td,
+			:host ::content > tbody > tr[active][selected] th,
+			:host([selectable]) ::content > tbody > tr[selected]:hover td,
+			:host([selectable]) ::content > tbody > tr[selected]:hover th,
+			:host ::content > tbody > tr[active][selected] + tr > td,
+			:host ::content > tbody > tr[active][selected] + tr > th,
+			:host([selectable]) ::content > tbody > tr[selected]:hover + tr > td,
+			:host([selectable]) ::content > tbody > tr[selected]:hover + tr > th {
+				border-top-color: var(--d2l-table-row-border-color-active-selected) !important;
+			}
+
+			:host ::content > tbody > tr:last-child[selected] td,
+			:host ::content > tbody > tr:last-child[selected] th {
+				border-bottom-color: var(--d2l-table-row-border-color-selected) !important;
+			}
+
+			:host ::content > tbody > tr:last-child[active][selected] td,
+			:host ::content > tbody > tr:last-child[active][selected] th,
+			:host([selectable]) ::content > tbody > tr:last-child[selected]:hover td,
+			:host([selectable]) ::content > tbody > tr:last-child[selected]:hover th {
+				border-bottom-color: var(--d2l-table-row-border-color-active-selected) !important;
+			}
+
+			:host ::content .d2l-table-col-sort-asc::after,
+			:host ::content .d2l-table-col-sort-desc::after {
+				display: inline-block;
+				vertical-align: middle;
+			}
+
+			:host ::content .d2l-table-col-sort-asc::after {
+				content: url('../d2l-icons/images/tier1/arrow-toggle-up.svg');
```

Comment:
I was thinking of doing something like this in another PR, since the goal is to get the table looking the same as before with WC. But yeah, if the assets won't necessarily be available...

I do plan on changing this before a release though. Are you okay with me opening an issue and resolving this later?